README.md

```diff
@@ -246,11 +247,84 @@ Your translation files:
 - Should be named with the same name as you set on the [config method](#install-and-setup) of the service you called in your `AppComponent`. The same rule goes for further `.setLocale(locale)` calls to change the language of your application.
 - Must always have `.i18n.json` as extension. (`en-us.i18n.json`)
 
-We recommend that you scope your translation keys inside a component key. This process will be automatized [in the future](#roadmap).
+We recommend that you scope your translation keys inside a component key. This process can be automatized using the [embedded CLI](#the-cli).
+
+### _- Configurable http loader_
+
+Instead of using the default assets loader, you can instead use the other embedded loader, the Http Loader. This uses a special configuration in `config` to give you complete freedom of where/how to store your locale data. To use, simply need to import the module at `root` using the `httpLoader()` method:
+
+```typescript
+import { Smarti18nModule } from 'smarti18n';
+
+// [...]
+
+@NgModule({
+  imports: [
+    Smarti18nModule.httpLoader()
+  ]
+})
+```
+
+and pass the necessary configuration, following the schema into the optional `loader` property on `setConfig()`:
+
+```typescript
+smarti18n.setConfig({
+	defaultLocale: 'en-us',
+	locale: 'pt-br',
+	loader: {
+		"baseUrl": "http://baseUrl",
+		"suffix": "a_suffix"
+	}
+})
+```
+
+either `baseUrl` and `suffix` are optional, but at least one of them should be set. The final url follows the logic
+```typescript
+`${baseUrl || ''}/${locale}/${suffix || ''}`
+```
+so for the example above, we get `http://baseUrl/{locale}/a_suffix`
 
 ### _- Write your own loader_
 
-_TO DO!_
+You can create your own loader by creating a new service class, inheriting from `LocaleLoaderService`:
+
+```typescript
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable, of as observableOf } from 'rxjs';
+import { ObjMap, LocaleLoaderService } from 'smarti18n';
+
+export class MyOwnLoaderService extends LocaleLoaderService {
+	constructor() { super(); }
+
+	public load(locale: string): Observable<ObjMap<string>> {
+		return ObservableOf({
+			key1: 'value',
+			key2: {
+				key3: 'other value'
+			}
+		});
+	}
+}
+```
+

```

Comment:
I did not get how writing your own loader works by reading this ._.
should be the returned object inside the observable be the locale object? Souldn't we have a paragraph describing this?